Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.py

```diff
@@ -0,0 +1,1549 @@
+        ScheduledCommand: Command, args, timeout and interval for CommandResults.
+    """
+
+    if "sample_id" not in args:
+        command_results = upload_sample_command(client, args)
+        sample_id = command_results.raw_response["id"]  # type: ignore[index]
+        args["sample_id"] = sample_id
+    else:
+        command_results = sample_state_get_command(client, args)
+
+    sample_state = dict_safe_get(command_results.raw_response, ["state"])
+    sample_id = args["sample_id"]
+    args_for_next_run = {"sample_id": sample_id, **args}
+
+    if sample_state == "succ":
+        command_results = get_sample_command(client, args)
+        return PollResult(
+            response=command_results,
+            continue_to_poll=False,
+        )
+
+    return PollResult(
+        response=command_results,
+        continue_to_poll=True,
+        args_for_next_run=args_for_next_run,
+    )
+
+
+def get_dbotscore(
+    api_score: int,
+    generic_command_name: str,
+    indicator_value: str,
+    reliability: str,
+) -> Common.DBotScore:
+    """Get XSOAR score for the file's / IP's / URL's / domain's disposition.
+
+    Args:
+        api_score (int): The API score.
+        generic_command_name (str): The generic command name for identify
+            witch command are used for the indicator type.
+        indicator_value (str): The command argument - the indicator.
+        reliability (str): The reliability that chosen.
+
+    Returns:
+        Common.DBotScore: DBot Score according to the disposition.
+    """
+    score = 0
+    if not api_score:
+        score = Common.DBotScore.NONE
+    elif api_score >= 85:
+        score = Common.DBotScore.BAD
+    elif api_score >= 50:
+        score = Common.DBotScore.SUSPICIOUS
+    elif api_score < 50:
+        score = Common.DBotScore.GOOD
+
+    return Common.DBotScore(
+        indicator=indicator_value,
+        indicator_type=generic_command_name,
+        integration_name="ThreatGrid",
+        reliability=reliability,
+        score=score,
+    )
+
+
+def reputation_command(
+    client: Client,
+    args: Dict[str, Any],
+) -> Union[List[CommandResults], CommandResults]:
+    """
+    Generic reputation command that returns information about Files/IPs/URLs/Domains.
+    Args:
+        client (Client): Cisco ThreatGrid API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+    Returns:
+        List[CommandResults]: Indicator for every file_hash
+    """
+    generic_command_name = args["command_name"]
+    command_args = argToList(args[generic_command_name])
+    reliability = args["reliability"]
+    sample_id = ""
+    score = 0
+    command_results = []
+
+    for command_arg in command_args:
+        response = client.search_submission(
+            query=command_arg,
+            state="succ",
+            sort_by="analyzed_at",
+        )
+        if response["data"]["current_item_count"] == 0:
+            return CommandResults(readable_output="Unknown")
```

Comment:
Instead of a simple Unknown string, please do return an indicator, with `dbotscore = DBotscore.UNKNOWN`